Dockerfile

```diff
@@ -60,6 +59,7 @@ RUN apt-get update \
     pgsql \
     zip \
   && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* \
```

Comment:
I get this command, but it makes sense only if you run `RUN apt-get update` before, right?